api/conf/urls/__init__.py

```diff
@@ -37,5 +37,5 @@
     path("v1/", include(versioned_paths)),
 ]
 
-if settings.ENVIRONMENT == "local":
-    urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+# if settings.ENVIRONMENT == "local":
+#     urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
Is this left in intentionally?

Reply:
Nope! It's unnecessary now with the ASGI handler approach for the static files and I just forgot to remove the commented old code :sweat_smile: 

```suggestion
```

Reply:
Done in https://github.com/WordPress/openverse/pull/3011/commits/10a4c47d25ea45a7c7baedc93fb022e7a1a79968.